main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"os"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+)
+
+type config struct {
+	InputFilePath  string          `env:"encrypted_file_path,file"`
+	OutputFilePath string          `env:"output_file_path,required"`
+	Passphrase     stepconf.Secret `env:"decrypt_passphrase,required"`
+}
+
+func failf(format string, args ...interface{}) {
+	log.Errorf(format, args...)
+	os.Exit(1)
+}
+
+func main() {
+	var cfg config
+	if err := stepconf.Parse(&cfg); err != nil {
+		failf("Could not create config: %v\n", err)
+	}
+	stepconf.Print(cfg)
+	outputFileAbsPath, err := pathutil.AbsPath(cfg.OutputFilePath)
+	if err != nil {
+		failf("Converting to absolute path failed, error: %#v", err)
+	}
+	if err := os.Remove(outputFileAbsPath); err != nil && !os.IsNotExist(err) {
```

Comment:
How `gpg` handles the case when the output file already exists, may we should not silently delete files?